samples/README.md

```diff
@@ -1,12 +1,14 @@
-## Using basic AzureML APIs
+## Azure ML samples
+These samples are short code examples for using the Azure Machine Learning service with the R SDK. If you are new to the R SDK, we recommend that you first take a look at the more detailed end-to-end [vignettes](../vignettes).
```

Comment:
`[vignettes](../vignettes)` this needs to be updated with a full https url otherwise it would fail at R CMD check with invalid URL error.

Reply:
Was this check run for the CRAN submission? Because the relative URL was also used in the vignettes readme & that was there before latest CRAN submission

Reply:
Yes. Here are the changes I made last time for addressing the invalid URL issues. Check out README.md.
https://github.com/Azure/azureml-sdk-for-r/pull/133/files

but your changes passed devtools::check_rhub(), we should be good.